package.json

```diff
@@ -30,70 +31,60 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
-    "codemirror": "5.3.0",
+    "codemirror": "^5.65.0",
     "d3": "^3.1.5",
-    "es6-promise": "2.3.0",
-    "fetch": "1.0.0",
-    "flux": "~2.0.2",
-    "fontawesome": "^4.3.0",
     "jquery": "1.11.3",
     "jquery-ui": "1.12.1",
     "nunjucks": "^3.2.3",
-    "react": "0.13.3",
-    "underscore": "^1.13.1"
+    "underscore": "^1.13.2"
   },
   "devDependencies": {
-    "@babel/cli": "^7.12.16",
-    "@babel/core": "^7.12.16",
-    "@babel/preset-env": "^7.12.16",
-    "@babel/preset-react": "^7.12.13",
-    "@babel/register": "^7.12.13",
+    "@babel/core": "^7.16.7",
+    "@babel/plugin-transform-react-jsx": "^7.16.7",
+    "@babel/preset-env": "^7.16.7",
     "@mozilla-protocol/core": "10.0.1",
-    "autoprefixer": "^9.8.6",
+    "autoprefixer": "^10.4.1",
     "babel-loader": "^8.2.2",
-    "babel-plugin-module-resolver": "^4.1.0",
-    "babelify": "5.0.4",
-    "browser-sync": "^2.26.7",
-    "chai": "3.2.0",
+    "browser-sync": "^2.27.7",
+    "chai": "^4.3.4",
     "chai-lint": "0.1.1",
-    "color-string": "^1.6.0",
-    "concurrently": "^5.0.0",
-    "copy-webpack-plugin": "^9.0.0",
-    "css-loader": "^5.2.6",
-    "cssnano": "^4.1.11",
-    "eslint": "^8.1.0",
+    "color-string": "^1.9.0",
+    "concurrently": "^7.0.0",
+    "copy-webpack-plugin": "^10.2.0",
+    "css-loader": "^6.5.1",
+    "cssnano": "^5.0.14",
```

Comment:
node and postcss upgrades: https://github.com/cssnano/cssnano/releases/tag/cssnano%405.0.0